python/lsst/ts/ATHexapod/mock_controller.py

```diff
@@ -0,0 +1,256 @@
+                                                 y=matched_command.group('y_value'),
+                                                 z=matched_command.group('z_value'),
+                                                 u=matched_command.group('u_value'),
+                                                 v=matched_command.group('v_value'),
+                                                 w=matched_command.group('w_value'))
+                        elif matched_command.group('cmd') in [b'SPI']:
+                            self.log.debug(f"Grabbing command {matched_command.group('cmd')}")
+                            await called_command(x=matched_command.group('x_value'),
+                                                 y=matched_command.group('y_value'),
+                                                 z=matched_command.group('z_value'))
+                        elif matched_command.group('cmd') in [b'VLS']:
+                            self.log.debug(f"Grabbing command {matched_command.group('cmd')}")
+                            await called_command(velocity=matched_command.group('velocity'))
+                        elif await called_command() is not None:
+                            self.log.debug(f"Sent {await called_command()}")
+                            writer.write(await called_command())
+                            await writer.drain()
+                    else:
+                        raise Exception(f"{matched_command.group('cmd')} is not implemented.")
+
+    async def real_position(self):
+        return f"X={self.x} Y={self.y} Z={self.z} U={self.u} V={self.v} W={self.w}\n".encode()
+
+    async def motion_status(self):
+        return f"0\n".encode()
+
+    async def controller_ready(self):
+        self.ready = True
+        return f"{chr(177)}\n".encode()
+
+    async def stop_all_axes(self):
+        pass
+
+    async def set_position(self, x, y, z, u, v, w):
+        self.log.debug("Setting position")
+        self.commanded_x = x.decode()
+        self.comanded_y = y.decode()
+        self.commanded_z = z.decode()
+        self.commanded_u = u.decode()
+        self.commanded_v = v.decode()
+        self.commanded_w = w.decode()
+        self.x = x.decode()
+        self.y = y.decode()
+        self.z = z.decode()
+        self.u = u.decode()
+        self.v = v.decode()
+        self.w = w.decode()
+
+    async def referencing_result(self):
+        return (f"X={self.reference_x} "
+                f"Y={self.reference_y} "
+                f"Z={self.reference_z} "
+                f"U={self.reference_u} "
+                f"V={self.reference_v} "
+                f"W={self.reference_w}\n".encode())
+
+    async def reference(self):
+        self.reference_x = 1
+        self.reference_y = 1
+        self.reference_z = 1
+        self.reference_u = 1
+        self.reference_v = 1
+        self.reference_w = 1
+
+    async def target_position(self):

```

Comment:
You have a number of these methods that return a formatted string when the name suggests they return numeric data. Using a name such as `format_xxx` would make your intentional clearer (e.g. `format_target`).

Reply:
I *strongly* encourage you to use self-consistent terminology. I.e. use "target" or "commanded" but not a mix of both.